modules/webbrowser/webhelper/CMakeLists.txt

```diff
@@ -24,11 +24,13 @@ set(WEBBROWSER_HELPER_SOURCES
 )
 
 # place helper in separate executable. CEF uses app name + helper by default
-set(CEF_HELPER_TARGET "inviwo_helper" CACHE INTERNAL "CEF_HELPER_TARGET")
-set(CEF_HELPER_OUTPUT_NAME "Inviwo Helper")
+set(CEF_HELPER_TARGET "cef_inviwo_helper" CACHE INTERNAL "CEF_HELPER_TARGET")
+set(CEF_HELPER_OUTPUT_NAME "CEF Inviwo Helper")
```

Comment:
Maybe this is a OSX thing, but space in the filename is itching my eyes.

Reply:
Probably a Linux eye you got there ;) Should be fine in 2020 I hope

Reply:
Having spaces there seems likes begging for problems... ;)